tests/unit/core/reducers/test_versions.js

```diff
@@ -91,6 +91,26 @@ describe(__filename, () => {
     ]);
   });
 
+  it('loads a version with license text', () => {
+    const slug = 'some-slug';
+    const licenseText = 'license text';
+    const version = {
+      ...fakeVersion,
+      license: {
+        ...fakeVersion.license,
+        text: licenseText,
+      },
+    };
+    const state = versionsReducer(
+      undefined,
+      loadVersions({ slug, versions: [version] }),
+    );
+
+    const storedVersion = getVersionsBySlug({ slug, state })[0];
```

Comment:
Can you rename to `firstSstoredVersion`? That would give a hint to see the `[0]`.